.github/ISSUE_TEMPLATE/4_Documentation_issue.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
With this we create an infinite recursion when trying to open an issue on `api-platform/docs` :loop:

Reply:
What do you recommend?

I overload this template in the documentation repository?

Reply:
https://github.com/api-platform/.github/pull/1#discussion_r330552745

Reply:
Not a big issue IMO.

Reply:
I mean it's still better than nothing.